TOOLS/dylib-unhell.py

```diff
@@ -28,6 +28,11 @@ def otool(objfile):
     output  = subprocess.check_output(command, shell = True, universal_newlines=True)
     return set(filter(partial(is_user_lib, objfile), output.split()))
 
+def get_rpaths_dev_tools(binary):
+    command = "otool -l %s | grep -A2 LC_RPATH | grep path | grep \"Xcode\|CommandLineTools\" | awk '{ print $2 }'" % binary
```

Comment:
I mean, the command right on top of this one in `otool()` has the same issue, so I won't require this to be changed here (I'd rather both get fixed at the same time at some point if needed), but I'm pretty sure this would start failing due to lack of quoting with the `%s` in case spaces in paths? Possibly also the awk might fail in that case, but I don't recall which part of the output $2 was again :) .